keepalived.sysext/create.sh

```diff
@@ -19,7 +19,7 @@ function populate_sysext_root() {
   local img_arch="$(arch_transform 'x86-64' 'amd64' "$arch")"
   img_arch="$(arch_transform 'arm64' 'arm64/v8' "$img_arch")"
 
-  local image="docker.io/${img_arch}/alpine:3.19"
+  local image="docker.io/${img_arch}/alpine:3.21"
```

Comment:
```suggestion
  local image="docker.io/alpine:3.21"
```
as we explicitly specify `--platform` on the `docker run`  command line. For some reason the original version even workd for x86-64 but breaks for arm64.

Reply:
Thanks for catching that. I updated it accordingly.